openapi/components/schemas/QuoteChangeOrder.yaml

```
@@ -233,6 +233,38 @@ properties:
     description: |-
       Specifies if the quotation invoice must include interim items only.
       If this value is set to `false`, all upcoming items are included.
+  usageSettings:
+    type:
+      - 'object'
+      - 'null'
+    description: Settings for items with metered billing.
```

suggestion: to help make it clear that these are usage settings
```suggestion
    description: Usage settings for items with metered billing.
```